scripts/apps/authoring/versioning/history/history.js

```diff
@@ -30,7 +30,9 @@ function HistoryController($scope, authoring, api, notify, desks, archiveService
                             $scope.openVersion($scope.last);
                         }
                     }
-                });
+                }, function(err) {
```

Comment:
Ingested items don't have history. Remove the Item History tab for ingested item.